egs/wsj/s5/utils/combine_data.sh

```diff
@@ -39,6 +39,26 @@ for dir in $*; do
   fi
 done
 
+# W.r.t. utt2uniq file the script has different behavior compared to other files
+# it is not compulsary for it to exist in src directories, but if it exists in 
+# even one it should exist in all. We will create the files where necessary
+num_utt2uniq_files=$(find $* -name "utt2uniq"|wc -l)
```

Comment:
this is dangerous due to .backup directories.  You should iterate instead, e.g. for d in $*; do [ -f $x/utt2uniq ] && echo yes; done | wc -l